.github/workflows/build-rpms.yaml

```diff
@@ -0,0 +1,80 @@
+name: Trigger Microshift upstream build
+
+on:
+  workflow_dispatch:
+    inputs:
+      okd-version:
+        default: "4.18.0-okd-scos.4"
+        description: okd release from https://quay.io/repository/okd/scos-release?tab=tags
+        type: string
+      action:
+        type: choice
+        description: what to do?
+        default: run-in-bootc-container
+        options: 
+        - build-upstream-rpms
+        - run-in-bootc-container
+jobs:
+  build:
+    strategy:
+      matrix:
+        #disabled arm runner because images for arm no available in okd payload- see USHIFT-5570
+        runners: [ubuntu-24.04]
+    name: build microshift upstream
+    runs-on: ${{ matrix.runners }}
+    steps:
+      - name: Check out microshift-upstream CI repository
+        uses: actions/checkout@v4
+
+      - name: Check out microshift repository
+        uses: actions/checkout@v4
+        with:
+            repository: openshift/microshift
+            path: microshift
+            ref: main      
+
+      - name: Add cwd to path for kubectl.
+        run: echo `pwd` >> $GITHUB_PATH
+
+      - name: Build Microshift RPMs and Container
+        shell: bash
+        run: |
+          # /dev/sdb1  is mounted as /mnt - disk space needed for the podman build containers layers.
+          sudo mkdir -p /mnt/tmp/output
+          sudo chown -R 1001:127 /mnt/tmp
```

Comment:
Please, explain the choice of uid and gid in a comment.

Reply:
in general its permissions to store podman image layers (nouser) but i dont think we longer need that since now we are running `podman build` as root.